vhdbuilder/packer/vhd-image-builder-mariner.json

```diff
@@ -464,7 +469,32 @@
     {
       "type": "shell",
       "inline": [
-        "sudo FEATURE_FLAGS={{user `feature_flags`}} BUILD_NUMBER={{user `build_number`}} BUILD_ID={{user `build_id`}} COMMIT={{user `commit`}} HYPERV_GENERATION={{user `hyperv_generation`}} CONTAINER_RUNTIME={{user `container_runtime`}} TELEPORTD_PLUGIN_DOWNLOAD_URL={{user `teleportd_plugin_download_url`}} ENABLE_FIPS={{user `enable_fips`}} SGX_INSTALL={{user `sgx_install`}} IMG_SKU={{user `img_sku`}} /bin/bash -ux /home/packer/pre-install-dependencies.sh",
```

Comment:
@cameronmeissner when you originally added multiple reboots 5 months ago, Mariner FIPS was not supported. we do build Mariner FIPS SKU now, hence separated pre-install, install-fips and install-deps with a reboot in between each